unittest/unit/find-missing-features.js

```diff
@@ -14,74 +14,30 @@
 
 'use strict';
 
-const assert = require('chai').assert;
+import {assert} from 'chai';
 
-const proxyquire = require('proxyquire').noCallThru();
-const sinon = require('sinon');
+import sinon from 'sinon';
 
-const tests = {
-  'api.AbortController': {},
-  'api.AbortController.signal': {},
-  'css.properties.font-family': {},
-  'javascript.builtins.Array': {},
-  'javascript.builtins.Error': {}
-};
-const bcd = {
-  api: {
-    AbortController: {
-      __compat: {},
-      dummy: {
-        __compat: {}
-      },
-      signal: {
-        __compat: {}
-      }
-    },
-    DummyAPI: {
-      __compat: {},
-      dummy: {
-        __compat: {}
-      }
-    }
-  },
-  css: {
-    properties: {
-      'font-family': {
-        __compat: {}
-      },
-      'font-face': {
-        __compat: {}
-      }
-    }
-  },
-  javascript: {
-    builtins: {
-      Array: {
-        __compat: {}
-      },
-      Date: {
-        __compat: {}
-      }
-    }
-  }
-};
+import {traverseFeatures, getMissing} from '../../find-missing-features.js';
 
-const {traverseFeatures, getMissing} = proxyquire(
-  '../../find-missing-features',
-  {
-    './tests.json': tests,
-    '../browser-compat-data': bcd
-  }
-);
+import bcd from './bcd.test.js';
 
-describe('find-missing', () => {
+describe('find-missing', async () => {
```

Comment:
An addition of `async` without any added `await` is suspect:

```suggestion
describe('find-missing', () => {
```